src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="single-fieldset">
+        <legend>Personal information</legend>
+
+        <label class="single-label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="single-label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="single-label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="single-label">
+          Full date of birth:
+          <input
+            type="date"
+            name="bday"
+          >
+        </label>
+
+        <label class="single-label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="single-fieldset">
+        <legend>Registration</legend>
+
+        <label class="single-label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="single-label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="single-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="single-label inline">Do you love cats?
+
+          <input
+            type="radio"
+            name="love_cats"
+            value="yes"
+          > Yes
+
+          <input
+            type="radio"
+            name="love_cats"
+            value="no"
+          > No
+
+        </label>
```

Comment:
```suggestion
        <div class="single-label">Do you love cats?
          <label>
            <input
              type="radio"
              name="love_cats"
              value="yes"
            > 
            Yes
          </label>
          
          <label>
            <input
              type="radio"
              name="love_cats"
              value="no"
            > 
            No
          </label>
        </div>
```
each input should be wrapped in label.
without it click on Do you love cats or benween inputs or eveb on No select first radio input
and with this refactoring will be redundant inline class